libs/barista-components/chart/src/heatfield/chart-heatfield.scss

```diff
@@ -19,7 +19,25 @@ $animation-delay: 40ms;
   transition: transform $animation-duration ease-in-out $animation-delay;
 }
 
-.dt-chart-heatfield-marker.dt-chart-heatfield-active {
+.dt-chart-heatfield-marker .dt-chart-heatfield-text {
+  font-size: 6px;
+  font-weight: bold;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  display: block;
+  color: white;
+}
+
+.dt-chart-heatfield-marker.dt-chart-heatfield-expanded
+  .dt-chart-heatfield-text {
+  font-size: 8px;
+  transform: scaleY(0.666666); // The parent it's scaled to 1.5 when active
```

Comment:
That will definitely lead to some weird rendering artifacts. First scaling up then down. does that look sharp when displayed?

Reply:
I didn't appreciate anything wrong during the implementation. However, I have to admit that I don't like it very much.
How could we prevent the child to ignore the parent scale?